### PR TITLE
DAOS-623 ci: Do not make zero-value to GitHub for checkpatch.

### PR DESCRIPTION
**Pull request by ashleypittman**
If there is no feedback don't report anything, rather than giving a
empty LGTM comment.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

**Comment by ashleypittman** [comment]
Tested as part of https://github.com/daos-stack/daos/pull/13351

File: github_checkpatch.py [hunk 1]
```diff
@@ -482,6 +482,9 @@ def post_review(self, review_input):
         score, event, comments, review_comment = \
             self.create_github_review(review_input, commit.sha)
 
+        if not comments and review_comment == "LGTM.  No errors found by checkpatch.":
+            return
+
```
Comment (daltonbohning) [inline github_checkpatch.py:487]: This seems like a hack. Couldn't we just not post the message to begin with?

Comment (daltonbohning) [reply github_checkpatch.py:487]: And what if the message changes for some reason?

Comment (ashleypittman) [reply github_checkpatch.py:487]: Yes, it's a hack.  The message comes from elsewhere in this file - on some codepaths the message is appended to.  IMO this code is end-of-life, medium term we should port checkpatch to GitHub actions and remove all this.

Comment (daltonbohning) [reply github_checkpatch.py:487]: Wow. I didn't realize this was a completely separate repo. Yes, agree we should EOL this.

**Review by jolivier23 — APPROVED** [review]

**Comment by jolivier23** [comment]
I seem to recall the reason for the comment was something like cleariing an older negative review

**Comment by ashleypittman** [comment]
> I seem to recall the reason for the comment was something like cleariing an older negative review. @brianjmurrell do you remember anything about this?

Oh that's possible.

**Comment by brianjmurrell** [comment]
> I seem to recall the reason for the comment was something like cleariing an older negative review. @brianjmurrell do you remember anything about this?

Yes, that sounds quite familiar.

TBH, I am skeptical that we want to actually do this.  The fact that the testing output is different than the most recent ones on master is concerning also.  While that testing on master is failing also (so, test is probably broken), the output looks different so I suspect this change is messing with the testing also.

I'd be quite hesitant to land this would first fixing the testing on master and then seeing how this PR does with that fixed testing.

I think some testing would need to be added (or at least done manually with removing this comment from a review posting as this PR is trying to do to ensure that the results are what we want.  I suspect they won't be.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Per my prior comments.

**Comment by ashleypittman** [comment]
I've got a better idea for how to approach this.

**Pull request closed without merging**